lingetic-nextjs-frontend/app/components/questions/FillInTheBlanks/Question.tsx

```
@@ -0,0 +1,86 @@
+      </div>
+      <p className="text-skin-base mb-4">Hint: {question.hint}</p>
+      {!isChecked && (
+        <button
+          disabled={isChecking}
+          onClick={() => {
+            void handleCheckAnswer();
+          }}
+          className={`bg-skin-button-primary text-skin-inverted px-4 py-2 rounded transition-colors ${
+            isChecking ? "opacity-70" : ""
+          }`}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)